datastore/Rakefile

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "rake/testtask"
+
+Rake::TestTask.new "test" do |t|
+  t.test_files = FileList["**/*_test.rb"]
+  t.options = "--junit --junit-filename=sponge_log.xml"
+  t.warning = false
+end
```

Comment:
@TheRoyalTnetennba Should there be an `acceptance ` task in this file? There is one in #489 but not in #542.

Reply:
I don't think the acceptance task is needed atm. Wasn't originally sure how the ci for samples was going to look

Reply:
Good, thanks!